go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/BurntSushi/toml v1.5.0 // MIT
 	github.com/Masterminds/semver/v3 v3.4.0 // MIT
 	github.com/briandowns/spinner v1.23.1 // Apache 2.0
-	github.com/databricks/databricks-sdk-go v0.89.0 // Apache 2.0
+	github.com/databricks/databricks-sdk-go v0.90.0 // Apache 2.0
```

Comment:
This needs to back out. SDK bumps are done centrally.

Reply:
done